src/Components/TEST/Data.js

```diff
@@ -0,0 +1,82 @@
+export const data = [
+    {
+      project: "Project-1",
+      daily: 5,
+      weekly: 20,
+      monthly: 50,
+      quarterly: 100,
+    },
+    {
+      project: "Project-2",
+      daily: 7,
+      weekly: 15,
+      monthly: 40,
+      quarterly: 20,
+    },
+    {
+      project: "Project-3",
+      daily: 3,
+      weekly: 10,
+      monthly: 30,
+      quarterly: 60,
+    },
+    {
+      project: "Project-4",
+      daily: 9,
+      weekly: 25,
+      monthly: 60,
+      quarterly: 120,
+    },
+  ];
+
+// IssueTypeBar 
```

Comment:
Please Write an elaborate comment here.....Not able to understand just by IssueTypeBar

Reply:
ok I will elaborate it.

Reply:
Changes Accepted